src/main/java/com/google/devtools/build/lib/sandbox/AbstractSandboxSpawnRunner.java

```diff
@@ -230,7 +230,11 @@ private boolean wasTimeout(Duration timeout, Duration wallTime) {
       throws IOException {
     // We have to make the TEST_TMPDIR directory writable if it is specified.
     ImmutableSet.Builder<Path> writablePaths = ImmutableSet.builder();
-    writablePaths.add(sandboxExecRoot);
+
+    if (OS.getCurrent() != OS.WINDOWS) {
+      writablePaths.add(sandboxExecRoot);
```

Comment:
Could you please add a comment that explains this conditional?
(I.e. that all windows-sandboxed spawns use the same execroot, which is safe because we specify exactly which files are readable/writable)